README.md

```diff
@@ -7,367 +7,598 @@
+
+#### `SUPERTAPE_PROGRESS_BAR_MIN`
+
+If using the `progress-bar` formatter, sets the minimum number of tests to run before showing the progress bar. Defaults to `100`. Set to `0` to always show.
+
+#### `SUPERTAPE_PROGRESS_BAR_STACK`
+
+If using the `progress-bar` formatter, sets a flag to show or hide the error stack when an assertion fails. Defaults to `1` (*show*). Set to `0` to hide.
+
+### `node` and `ts-node`
+
+Instead of using the command line, any test file (`.js` or `.ts`) requiring/importing 📼 **Supertape** can be invoked through `node` or `ts-node`, respectively:
 
 ```js
-test('tape: error', (t) => {
-    t.equal(error.code, 'ENOENT');
+// test.js or test.ts
+import test from 'supertape';
+
+test('test: from (ts-)node', (t) => {
+    t.pass('hello, world!');
     t.end();
 });
 ```
 
-## 🤷 How to migrate from **tape**?
+```bash
+$ node test.js
+# OR
+$ ts-node test.ts
+```
 
-> 🐊 + 📼 = ❤️
+Options can be specified either by using [environment variables](#environment-variables) or on a per-file basis with `test.init(options)` or `test(message, fn, options)`.
 
-You can convert your codebase from **tape** to 📼**Supertape** with help of 🐊[**Putout**](https://github.com/coderaiser/putout), which has built-in [**@putout/plugin-tape**](https://github.com/coderaiser/putout/tree/master/packages/plugin-tape#readme),
-with a lot of rules that helps to write and maintain tests of the highest possible quality.
+#### **<u>With Environment Variables</u>:**
 
-Here is [result example](https://github.com/coderaiser/cloudcmd/commit/74d56f795d22e98937dce0641ee3c7514a79e9e6).
+For example, setting options for the `progress-bar` formatter:
 
-## **ESLint** rules
+```bash
+$ SUPERTAPE_PROGRESS_BAR_MIN=10 node test.js
+```
 
-[**eslint-plugin-putout**](https://github.com/coderaiser/putout/tree/master/packages/eslint-plugin-putout#-supertape) has a couple rules for 📼**Supertape**:
+#### **<u>Initialization</u>:**
 
-- ✅ [`remove-newline-before-t-end`](https://github.com/coderaiser/putout/tree/master/packages/eslint-plugin-putout/lib/tape-remove-newline-before-t-end#readme)
-- ✅ [`add-newline-before-assertion`](https://github.com/coderaiser/putout/tree/master/packages/eslint-plugin-putout/lib/tape-add-newline-before-assertion#readme)
-- ✅ [`add-newline-between-tests`](https://github.com/coderaiser/putout/tree/master/packages/eslint-plugin-putout/lib/tape-add-newline-between-tests#readme)
+📼 **Supertape** also provides a function `test.init(options)` that initializes the test engine with the specified options:
 
-## Validation checks
+```js
+const {init} = require('supertape');
 
-To up the quality of your tests even higher, 📼**Supertape** has built-in checks.
-When test not passes validation it marked as a new failed test.
+init({
+    format: 'json-lines',
+});
```

Comment:
I don’t think this going to work this way, formatters is set from CLI.

Reply:
It's the only way to set `format` when using `node`:

![image](https://user-images.githubusercontent.com/1403701/188556891-efe129cd-2732-4220-9c63-bd05782070da.png)

Reply:
Yes, I didn't realize it was internal usage because it's included on the `test` object, as well as `createStream()`:

![image](https://user-images.githubusercontent.com/1403701/188559468-1b481bad-6d6b-4fac-b2fc-595004381d0f.png)

Reply:
Well it used from the CLI only... OK, it's a good way to init options for all tests :).